trustmanager/keyfilestore_test.go

```diff
@@ -10,9 +11,18 @@ import (
 	"testing"
 )
 
+var passphraseRetriever = func(keyID string, alias string, createNew bool, numAttempts int) (string, bool, error) {
```

Comment:
Need a test with a passphraseRetriever that only returns errors to test attempt cap.